packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -201,22 +201,22 @@ export class FluidDataStoreRuntime
 		return this.dataStoreContext.idCompressor;
 	}
 
-	public get IFluidHandleContext() {
+	public get IFluidHandleContext(): this {
```

Comment:
Note: this type is exported by the package, so changing the return types here to align with the base interface is, strictly speaking, a breaking change (and is caught by our type-tests). So, I have opted to leave the typing here alone.

Reply:
Add a TODO?

Reply:
```suggestion

	// TODO: the methods below should have more specific return typing, per the interfaces they are implementing.
	// Doing so would be a breaking change.

	public get IFluidHandleContext(): this {
```